app/waf/src/datasources/as3.datasource.json

```diff
@@ -0,0 +1,21 @@
+{
+  "connector": "rest",
+  "options": {
+    "headers": {
+      "accept": "application/json",
+      "content-type": "application/json"
+    },
+    "strictSSL": false
+  },
+  "operations": [
+    {
+      "template": {
```

Comment:
can we add a timeout property? 
ping for as3 information should be over very quickly. waiting for default 2min timeout may be too long..

Reply:
I do not find a way to specify timeout for outbound calls. Do you know how to specify it?

Reply:
Don't know yet.